tests/test_meadowflow/test_scheduler.py

```diff
@@ -599,7 +600,7 @@ def _run_instantiate_date_scope() -> ScopeValues:
 
 
 @add_scope_jobs_decorator
-def _run_add_scope_jobs(scope: ScopeValues) -> Sequence[Job]:
+def _run_add_scope_jobs(scope: ScopeValues, /) -> Sequence[Job]:
```

Comment:
Is this needed? I think I was able to take it out without causing any problems

Reply:
No - I had a hard time typing that decorator so was trying all sorts of stuff.